packages/@sfcc-core/apiconfig/package.json

```diff
@@ -1,20 +1,44 @@
 {
-  "name": "@sfcc-core/apiconfig",
-  "version": "1.0.0-alpha.1",
-  "description": "sfcc backend for frontend api service",
-  "main": "./src/apiconfig.js",
-  "type": "module",
+  "license": "BSD-3-Clause",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "lint": "eslint src",
-    "lint:fix": "eslint src --fix",
-    "pak": "npm pack",
-    "pub": "npm pack; npm publish -f",
-    "link": "npm link",
-    "unlink": "npm link"
+    "start": "tsdx watch",
```

Comment:
We are using the industry standard tsdx (https://github.com/jaredpalmer/tsdx) for doing most of the TypeScript work.